### DISCUSSION TITLE
OpenRadioss for dummies

### DISCUSSION THREAD

**Discussion by Bertrand53440** [OpenRadioss/OpenRadioss#1017] [Q&A] [answered]
Hi,
i am looking for the best way to use OpenRadioss using only opensources softwares... so not possible to use Hypermesh to built the model.

so i am used to use Freecad, Gmsh paraview.... 

and I am wondering how to build the input file for Openradioss... where can I find a basic tutorial to start from the CAD part, to mesh it, to defined the rest of the model (BC, Materials, loding case etc...).
I didn't find a GUI able to help me in this case.... (as i use for others types of solvers : Prepomax, Simflow, freecad etc..)

thank for your help.

bertrand

**Comment by PaulAltair** [comment, marked as answer]
The latest version of GMSH (development build) allows export of Radioss mesh format: There is a very simple example in the OpenRadioss confluence pages: [Tensile Test Example using Gmsh](https://openradioss.atlassian.net/wiki/spaces/OPENRADIOSS/pages/24444938/Tensile+Test+Example+Tutorial+Using+Gmsh)
BCS/Materials/Properties are pre-defined in this example, there is not a 'free' GUI tool for these as yet that I know of

**Comment by Bertrand53440** [comment]
ok. I saw that. nice to be able to use gmsh.

I want to use on win.. so I download the winx64 zip file. and now i am looking for procedure to use openradioss on win (built ? compiling ? environnement variables ??)
